File: src/main/java/com/ultreon/mods/motobox/Motobox.java
package com.ultreon.mods.motobox;

import com.ultreon.mods.motobox.items.ModItems;
import net.fabricmc.api.ModInitializer;
import net.minecraft.util.Identifier;

public class Motobox implements ModInitializer {
    private static final String MOD_ID = "motobox";

    public static Identifier id(String path) {
        return new Identifier(MOD_ID, path);
    }

    @Override
    public void onInitialize() {
        ModItems.init();
    }
}

File: src/main/java/com/ultreon/mods/motobox/Motobox.kt
package com.ultreon.mods.motobox

import com.ultreon.mods.motobox.items.ModItems.init
import net.fabricmc.api.ModInitializer
import net.minecraft.util.Identifier

object Motobox : ModInitializer {
    private const val modId = "motobox"

    override fun onInitialize() {
        init()
    }

    @JvmStatic
    fun id(path: String?): Identifier {
        return Identifier(modId, path)
    }
}